src/Sylius/Bundle/ApiBundle/ApiPlatform/Symfony/EventListener/WriteListener.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\ApiPlatform\Symfony\EventListener;
+
+use ApiPlatform\Api\IriConverterInterface as LegacyIriConverterInterface;
+use ApiPlatform\Api\ResourceClassResolverInterface as LegacyResourceClassResolverInterface;
+use ApiPlatform\Api\UriVariablesConverterInterface as LegacyUriVariablesConverterInterface;
+use ApiPlatform\Exception\InvalidIdentifierException;
+use ApiPlatform\Metadata\Error;
+use ApiPlatform\Metadata\Exception\InvalidUriVariableException;
+use ApiPlatform\Metadata\HttpOperation;
+use ApiPlatform\Metadata\IriConverterInterface;
+use ApiPlatform\Metadata\Resource\Factory\ResourceMetadataCollectionFactoryInterface;
+use ApiPlatform\Metadata\ResourceClassResolverInterface;
+use ApiPlatform\Metadata\UriVariablesConverterInterface;
+use ApiPlatform\Metadata\Util\ClassInfoTrait;
+use ApiPlatform\Metadata\Util\CloneTrait;
+use ApiPlatform\State\CallableProcessor;
+use ApiPlatform\State\Processor\WriteProcessor;
+use ApiPlatform\State\ProcessorInterface;
+use ApiPlatform\State\UriVariablesResolverTrait;
+use ApiPlatform\State\Util\OperationRequestInitiatorTrait;
+use ApiPlatform\Symfony\Util\RequestAttributesExtractor;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Event\ViewEvent;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * This class is being overridden due to an issue with the clone method in the Adjustment class.
+ * The fix has been merged in PR #6367 (https://github.com/api-platform/core/pull/6367), and we are waiting for the new version of ApiPlatform to be released.
+ *
+ * Bridges persistence and the API system.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ * @author Baptiste Meyer <baptiste.meyer@gmail.com>
+ */
+final class WriteListener
```

Comment:
Sth has gone wrong with the rebase probably and this one should be already removed but it is added again 🙈